jetty-ee9/jetty-ee9-jaas/src/main/java/org/eclipse/jetty/ee9/jaas/spi/PropertyFileLoginModule.java

```diff
@@ -97,7 +98,8 @@ private void setupPropertyUserStore(Map<String, ?> options)
                 }
             }
             PropertyUserStore newStore = new PropertyUserStore();
-            newStore.setConfig(filename);
+            ResourceFactory resourceFactory = ResourceFactory.of(newStore);
+            newStore.setConfig(resourceFactory.newResource(filename));
```

Comment:
`PropertyUserStore` should accept the string and do that resource factory logic within the `setConfig(String)` method.